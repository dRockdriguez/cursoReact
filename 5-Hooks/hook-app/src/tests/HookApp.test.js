import { shallow } from "enzyme";
import { HookApp } from "../HookApp";
import React from "react";

describe("TEST HookApp", () => {
  test("Debe renderizarse", () => {
    const wrapper = shallow(<HookApp />);

    expect(wrapper).toMatchSnapshot();
  });
});
