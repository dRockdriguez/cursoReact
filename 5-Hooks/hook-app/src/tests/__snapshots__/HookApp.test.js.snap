// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TEST HookApp Debe renderizarse 1`] = `
<Fragment>
  <h1>
    Hola
  </h1>
</Fragment>
`;
